scripts/llvm_cfg_gen.py

```diff
@@ -8,7 +8,7 @@
     "to_substitute" : ["malloc", "free", "realloc", "calloc", "lp_id_get", "lib_ctx_get"],
     "substituted_suffix" : "_mt",
     "processed_suffix" : "_pr",
-    "to_ignore" : ['_log_log']
+    "to_ignore" : ["_log_log", "__acrt_iob_func"]
```

Comment:
Ugh! :scream:
This again makes me wonder if we really want to rewrite all functions...

Reply:
___acrt_iob_func()_ is an internal Microsoft API used to implement the C standard library
https://docs.microsoft.com/en-us/cpp/c-runtime-library/internal-crt-globals-and-functions?view=msvc-160#internal-crt-functions-and-function-macros
In particular, it is used to give access to the standard C streams (stdout, stderr, stdin):
https://stackoverflow.com/a/30894349
For some reason, the TargetLibraryInfoWrapperPass, used in the LLVM plugin to distinguish and "discard" system library functions, doesn't flag that function correctly.
Sadly, we would have the same problem even if we only instrumented the model code.

Reply:
:cry: